src/js/components/report/report-jump-nav.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-row class="jump-nav-bar">
+    <v-col xs="12">
+      <div>
+        <v-tabs v-model="activeTab" background-color="primary" show-arrows dark>
+          <v-tab v-scroll-to="{ el: '#summary', offset: -150, cancelable: false }" tag="button" rounded depressed>
+            Summary
+          </v-tab>
+          <v-tab
+            v-for="category in visibleCategories"
+            :key="formatAnchor(category)"
+            v-scroll-to="{ el: `#${formatAnchor(category)}`, offset: -60, cancelable: false }"
+            tag="button"
+            rounded
+            depressed
+          >
+            {{ category }}
+          </v-tab>
+        </v-tabs>
+      </div>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+import { mapGetters } from "vuex";
+
+export default {
+  name: "ReportJumpNav",
+  data() {
+    return {
+      activeTab: null,
+    };
+  },
+  created() {
+    this.$store.subscribe((mutation) => {
```

Comment:
Hmm.. I wouldn't use `$store.subscribe()` here -- since `activeCategory` is coming in via the getters anyway as a computed property, it's probably cleaner to make `activeTab` also a computed property. I'll push a commit that does that so you can see what I'm talking about.

Reply:
Oh nice! I haven't seen it done way you updated it before.